components/app_bar/index.d.ts

```diff
@@ -31,6 +43,26 @@ interface AppBarProps extends __ReactToolbox.Props {
    */
   flat?: boolean;
   /**
+   * If it exists it is used as the AppBar title
+   */
+  title? string;
```

Comment:
Hi, you probably just missed it but there should be a `:` between the property and type ( `title?: string`), like you did above.